doc/radosgw/compression.rst

```diff
@@ -103,3 +103,12 @@ size of compressed data, in bytes and kilobytes respectively.
 
 .. _`Server-Side Encryption`: ../encryption
 .. _`Multisite Configuration`: ../multisite
+
+Compression Tuning
+==================
+
+The compression level and window size is controlled by the ceph configuration parameters::
+
+  compressor_zlib_level
+  compressor_zlib_winsize
+  compressor_zstd_level
```

Comment:
can use 'confval' to make these render like the variables in https://docs.ceph.com/en/latest/radosgw/config-ref/#confval-rgw_frontends
```
.. confval:: compressor_zlib_level
```

Reply:
I'll take care of it. Thanks, Casey.

Reply:
Well, confval would work if there were confval-related information in doc/src/common/options/rgw.yaml.in.

Reply:
these are in global.yaml.in

Reply:
I'm going to resolve this, I was looking at an old version of the documents when I thought "gee we should list the valid types" but didn't notice when I was preparing the PR that we had a list now.